docs/use-case-descriptions/message-validation.md

```diff
@@ -13,25 +13,73 @@ If it fails validation, a response message is sent with ‘rejected’ in the re
+| Unknown SenderDomain | SignedMessage\SenderDomain doesn't match SignedMessage\Body\SenderDomain. | SM & All other message types |
+| Barred Sender | This endpoint is explicitly blocking messages from this sender. | SM |
+| Duplicate Identifier | The MessageID attribute of the inner XML message is not unique, and has already been used for a message with different content. This message has been rejected. | All message types except SM |
+| Already Submitted | The MessageID attribute of the inner XML message is not unique, but since the message content is the same as that of a previously accepted message, this copy can be considered to be successfully submitted as well. | All message types except SM |
+| ISP duration rejected | The message specifies a ISP duration that is not the agreed-upon common value for the market in which it is used. | All message types that contain ISP's: FRU, DP, FR, FO, FOR, M, FS, FS_R|
+| TimeZone rejected | The message specifies a TimeZone that has a different UTC offset than is the agreed-upon common value for the market. | All message types that contain Period: APQ, APQ_R, DPQ, DPQ_R, FRU, DP, FR, FO, FOR, M, FS, FS_R |
+| Invalid CongestionPoint | Unknown CongestionPoint or the recipient is not active at this CongestionPoint. | FRU, DP, FR, FO, FOR, FS |
+| Unknown <MessageType\>MessageID reference | The message the MessageID refers to is unknown.</br>For <MessageType\> the referenced MessageType can be filled in.</br>For example: *FlexRequestMessageID* or *D-PrognosisMessageID*. | APU_R, APQ_R, FRU_R, DP_R, FR_R, FO, FOF_R, FOFR, FOFR_R, FOR, FOR_R, M_R, FS, FS_R |
+| Reference Period mismatch | The message the <MessageType\>MessageID refers to contains a different Period. | MessageTypes that refer to other messages using <MessageType\>MessageID AND contain a Period themselves: APQ_R, FO FOFR, FOR, FS, FS_R |
+| Reference message expired | The message <MessageType\>MessageID refers to is expired. | APU_R, APQ_R, FRU_R, DP_R, FR_R, FO, FOF_R, FOFR, FOFR_R, FOR, FOR_R, M_R, FS, FS_R |
+| Reference message revoked | The message <MessageType\>MessageID refers is revoked. | APU_R, APQ_R, FRU_R, DP_R, FR_R, FO, FOF_R, FOFR, FOFR_R, FOR, FOR_R, M_R, FS, FS_R |
+| ISPs out of bounds | One or more ISPs are outside the tolerated boundaries.</br> - At least 1 ISP should exist in the messages that expect them;</br>- Also, the amount of ISPs that are about a Period (one *day*) can never exceed *(24 hours / ISP-Duration (in hours)) + summer-_wintertime_offset*</br>What that means is that for example: you can't have more than (24 hours (1 Period) / 0.25 (PT15M)) = 96 ISPss;</br>An exception is that during summer/wintertime (for Europe for example) you have an hour less or more on the day so the maximums would in that case be: 96 - 4 = 92 or 96 + 4 = 100. More ISPs for that specific Period would be invalid. | FRU, DP, FR, FO, FOR, M, FS, FS_R |
```

Comment:
Preferably use the terms Daylight Saving time (DST) instead of summertime/ wintertime

Reply:
Good suggestion about the DST, I'll use that.

In my MarkDown preview in VSCode the br tags work; I looked into alternatives of two spaces and an Enter and backslashes but then my table gets messed up.

Reply:
Do you experience issues with the br tag?

Reply:
no, but I am worried that it might not be really portable; if it works for you then it's ok I guess